core/services/vrf/metrics.go

```diff
@@ -1,119 +0,0 @@
-package vrf
```

Comment:
moved somewhere else?

Reply:
Yup, moved to `core/services/vrf/vrfcommon/metrics.go`